exporters/autoexport/exporter_test.go

```diff
@@ -20,41 +20,37 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/sdk/trace"
 )
 
-func TestOTLPExporterIsReturnedWhenNoEnvOrFallbackExporterIsConfigured(t *testing.T) {
+func Test_otlp_exporter_returned_when_no_env_or_fallback_exported_configured(t *testing.T) {
```

Comment:
nit: is there a reason the test names were changed from pascal to snake case? From what I can tell the former is more common throughout this repo.

Reply:
I found them much easier to read with such long names. I can revert if necessary but with them being test names, I thought clarity was important.

Reply:
Reverted test names to PascalCase.